lib/stm32/f4/adc.c

```diff
@@ -406,13 +406,13 @@ bool adc_awd(uint32_t adc)
 /*---------------------------------------------------------------------------*/
 /** @brief ADC Enable The Temperature Sensor
 
-This enables both the sensor and the reference voltage measurements on channels
-16 and 17. These are only available on ADC1 channel 16 and 17 respectively.
-
-@param[in] adc Unsigned int32. ADC block register address base @ref adc_reg_base
```

Comment:
Thanks for this,  I saw it in the context of your earlier diff and was going to strip it out when I got to it.